src/Products/Annotation/Controllers/AnnotationApiController.cs

```diff
@@ -592,145 +465,38 @@ public HttpResponseMessage Annotate(AnnotationPostedDataEntity annotateDocumentR
-        /// </summary>
-        /// <param name="documentGuid">string</param>
-        /// <param name="documentType">string</param>
-        /// <returns>AnnotationInfo[]</returns>
-        private AnnotationInfo[] GetAnnotations(string documentGuid, string documentType, string password)
-        {
-            try
-            {
-                FileStream documentStream = new FileStream(documentGuid, FileMode.Open, FileAccess.Read, FileShare.ReadWrite);
-                DocumentType docType = DocumentTypesConverter.GetDocumentType(documentType);
-                AnnotationInfo[] annotations = new BaseImporter(documentStream, AnnotationImageHandler, password).ImportAnnotations(docType);
-                documentStream.Dispose();
-                documentStream.Close();
-                return annotations;
-            }
-            catch (System.Exception ex)
-            {
-                throw ex;
-            }
-        }
+        public void RemoveAnnotations(string documentGuid, string password) {
+            // TODO: consider global properties
+            string tempFilename = Path.GetFileNameWithoutExtension(documentGuid) + "_tmp";
+            string tempPath = Path.Combine(Path.GetDirectoryName(documentGuid), tempFilename + Path.GetExtension(documentGuid));
 
-        private List<PageDataDescriptionEntity> GetAnnotatedPagesForPrint(string path)
-        {
-            AnnotatedDocumentEntity description = new AnnotatedDocumentEntity();
             try
             {
-
-                Stream document = File.Open(path, FileMode.Open, FileAccess.Read, FileShare.ReadWrite);
-                List<PageImage> pageImages = AnnotationImageHandler.GetPages(document, new ImageOptions());
-                document.Dispose();
-                document.Close();
-                for (int i = 0; i < pageImages.Count; i++)
+                using (Stream inputStream = File.Open(documentGuid, FileMode.Open, FileAccess.ReadWrite, FileShare.ReadWrite))
                 {
-                    PageDataDescriptionEntity page = new PageDataDescriptionEntity();
-                    if (pageImages != null)
+                    using (GroupDocs.Annotation.Annotator annotator = new GroupDocs.Annotation.Annotator(inputStream, GetLoadOptions(password, true)))
                     {
-                        byte[] bytes;
-                        using (MemoryStream memoryStream = new MemoryStream())
-                        {
-                            Stream imageStream = pageImages[i].Stream;
-                            imageStream.Position = 0;
-                            imageStream.CopyTo(memoryStream);
-                            bytes = memoryStream.ToArray();
-                        }
-                        string encodedImage = Convert.ToBase64String(bytes);
-                        page.SetData(encodedImage);
+                        annotator.Save(tempPath, new SaveOptions() { AnnotationTypes = AnnotationType.None });
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove these redundant parentheses.](https://app.codacy.com/manual/GroupDocs/GroupDocs.Annotation-for-.NET-MVC/pullRequest?prid=5372470)